### ISSUE TITLE
specify putio subfolder

### ISSUE THREAD

**Issue by tgianko** [datashaman/putio-automator#2]
Hi,

I stumbled upon your tool and I found it very useful! 

However, I cannot figure out a way to limit putio-automator to a putio subfolder. Basically, I have several put.io subfolders and I'd like to isolate putio-automator to, for example, automator/. Any idea how can I do that?

Thanks

**Comment by datashaman** [comment]
​Hi,

You control which folders are used in config.​py. Do you want a way to
specify those things when running the command instead?

M.

**Comment by tgianko** [comment]
That's great!  Which global variable should I change then? As far as I understood these three:

```python
INCOMPLETE = '/somefolder/Incomplete'
TORRENTS = '/somefolder/Torrents'
DOWNLOADS = '/somefolder/Downloads'
```

are for local folder. Or do they also refer to put.io subfolder?

**Comment by datashaman** [comment]
Aaaahhhhh, I understand what you want now. You want to only operate in a
specific folder on put.io, instead of the root folder. Got it.

I've just pushed a change to the files_list and files_download tasks so you
can specify a parent_id. This is the numeric ID at the end of the file
folder URL. If you don't specify one, it defaults to 0, which is the root
folder.

Have fun!

**Comment by tgianko** [comment]
This is really great, thanks! Do you think that the same thing can be done with `torrents_add` and `torrents_watch`?

**Comment by datashaman** [comment]
Hmm, lemme check if the putio API supports it...

**Comment by datashaman** [comment]
Looks like you're in luck, I'll push up another change now.

**Comment by datashaman** [comment]
I've pushed to develop branch, can you switch to that and test it for me? If it works correctly, I'll push out another release.

**Comment by tgianko** [comment]
Great! 

So, I tested `files_list` and `files_download`. They work pretty well. For the other two, I will have sometime today. I will let you know the result.

 Just one minor comment for `files_download`. I noticed that it is a bit hard to assess what is happening. Would it be possible to send some feedback to the stdout? E.g., which file is retrieving, which file is deleting and so on. I plan to run this in a server and I'd like to keep logs.
